Alicerce/Sources/Logging/Formatters/LogItemLevelColorBashFormatter.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  LogItemLevelColorBashFormatter.swift
+//  Alicerce
+//
+//  Created by Meik Schutz on 04/04/17.
+//  Copyright © 2017 Mindera. All rights reserved.
+//
+
+final class LogItemLevelColorBashFormatter {
+
+}
+
+extension LogItemLevelColorBashFormatter: LogItemLevelColorFormatter {
+    
+    internal var reset: String { return "\u{001b}[0m" }
```

Comment:
The compiler probably optimizes this, but I try to define constant properties as `let x = "something"` instead of a computed property